src/05-derivatives/03-imaging.md

```diff
@@ -131,13 +131,13 @@ and the `Atlas` metadata SHOULD be defined.
 
 JSON metadata fields:
 
-| **Key name** | **Description**                                                                                                                                |
-| ------------ | ---------------------------------------------------------------------------------------------------------------------------------------------- |
-| RawSources   | Same as defined in [Introduction][intro], but elevated from OPTIONAL to REQUIRED                                                               |
```

Comment:
@effigies could this be an error here? RawSources is defined in `02-common-data-types.md`, but not in `01-introduction.md`.

Reply:
Yes, looks like it broke during a reorganization.